examples/helm-template/helloworld-chart/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
```

Comment:
these should be used in the tests:

https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/tests/e2e.sh


We should create conventions for where each of the following lives for each function in this repo:

1. Function Source
2. Function Examples
3. Functions e2e tests

Reply:
Test helm template using kpt in e2e script now.